src/style/color.rs

```diff
@@ -141,14 +140,44 @@ impl Color {
     }
 }
 
+#[cfg(feature = "serde")]
+impl serde::Serialize for Color {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        serializer.serialize_str(&self.to_string())
+    }
+}
```

Comment:
Would it be worth adding docs to this to highlight that the serialization format is the same as the Display format?
This (or the deserialization code) would be a place where it's worth noting the compatibility with previous formats.